CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.0.11]
+
+### Added
+* Added to xyz_reader the ability to handle whitespace at the
```

Comment:
```suggestion
* Added to xyz_reader the ability to handle whitespaces at the
```

Reply:
I think this is fine as it is. Whitespace can refer to one or more spaces. 

https://en.wikipedia.org/wiki/Whitespace_character

>In computer programming, whitespace is any character or **series of characters** that represent horizontal or vertical space in typography.